Yaklin2 Onboarding

picoHacking opened the pull request:
Circuit to control car horns. Includes schematic, layout, and specification sheet.
DFM check: https://instantdfm.bayareacircuits.com/static/media/48a949r8_22699/work/reports/I8QuotationReport_48a949r8.pdf

picoHacking committed:
Onboarding for Illini Solar Car electrical team

Hello world LED circuit schematic.

picoHacking committed:
Merge branch 'yaklin2_onboarding' of https://github.com/IlliniSolarCar/ElecOnboarding25 into yaklin2_onboarding

picoHacking committed:
completed schematic and layout for the LED onboarding project

picoHacking committed:
Horns Board (Challenge) Project

Board to control the horns on the car. Includes schematic, layout, and specifications sheet.

picoHacking committed:
Horns Board + gbr files

Added .zip folder of gbr files.

picoHacking committed:
Update horns_board.kicad_pcb

Additional changes arising from the DFM setup.

picoHacking committed:
Completed introductory firmware project

This adds one LED (myLED or led_5), and turns it on and off at 1 Hz.

anav512 reviewed (commented):
Looks pretty good.
Firmware is fine. 
Hardware:
1. some footprint error.
2. recommended to modularize the schematic, rather than fully connected

picoHacking committed:
Updated according to comments

Split schematic into multiple modules instead of a single loop.
Replaced Molex kk254 connectors with micro fit and rerouted the traces accordingly.
